packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -105,36 +73,44 @@ storiesOf('Button', module)
     () => renderStyles()
   );
 
-function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T> = {variant: 'primary'}) {
-  return (
+function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T> & {showIconWithText?: boolean} = {variant: 'primary'}) {
+  let {
+    showIconWithText,
+    ...otherProps
+  } = props;
+  let buttonProps = mergeProps(otherProps, actions);
+
+  let buttons = (
     <Flex gap="size-200">
-      <Button
-        onPress={action('press')}
-        onPressStart={action('pressstart')}
-        onPressEnd={action('pressend')}
-        {...props}>
-        Default
+      <Button {...buttonProps}>
+        {showIconWithText && <Bell />}
+        <Text>Default</Text>
       </Button>
-      <Button
-        onPress={action('press')}
-        onPressStart={action('pressstart')}
-        onPressEnd={action('pressend')}
-        isDisabled
-        {...props}>
-        Disabled
+      <Button {...buttonProps} isDisabled>
+        {showIconWithText && <Bell />}
+        <Text>Disabled</Text>
       </Button>
       {props.variant !== 'cta' && (
-      <Button
-        onPress={action('press')}
-        onPressStart={action('pressstart')}
-        onPressEnd={action('pressend')}
-        isQuiet
-        {...props}>
-        Quiet
-      </Button>
+        <Button {...buttonProps} isQuiet>
+          {showIconWithText && <Bell />}
+          <Text>Quiet</Text>
+        </Button>
       )}
+      <Button {...buttonProps}>
```

Comment:
Can you add an `aria-label` here? I see an error in the accessibility tab:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/19409/189250680-ed9869ad-1928-4491-badc-f47aef94c069.png">